### DISCUSSION TITLE
Community Weekly Meeting (December 15, 2021)

### DISCUSSION THREAD

**Discussion by juzhiyuan** [apache/apisix#5786] [General]
Hi, here is the schedule for the coming Weekly Meeting.

**Schedule**

- **Date**: on Wednesday, December 15, at 20:00 ~ 21:30 (UTC +8)
- **Meeting Link**: https://meeting.tencent.com/dm/9OQXqMrWESZX
- **Topics**: Please leave your comment below :)

**Note**
- If you have any questions, please comment below or contact **juzhiyuan@apache.org**
- Kindly let me know if you need **Google Meet** or **Zoom** as communication tools.

**Previous Meeting**
- https://github.com/apache/apisix/discussions/5721

**Comment by juzhiyuan** [comment]
For tomorrow's meeting, I just created a doc to track topics and other contents: https://docs.qq.com/doc/DTkhKZFJtVHB5VXZ1

This doc is in Chinese, kindly let me know if you need it in English 😄

**Comment by juzhiyuan** [comment]
# Meeting Summary

> Doc: [Chinese Version](https://docs.qq.com/doc/DTkhKZFJtVHB5VXZ1)
> Video: https://www.bilibili.com/video/BV1aM4y1F7h7/

## Integrate with [arkid](https://github.com/longguikeji/arkid)

Q: https://github.com/apache/apisix/discussions/5778
A:
- https://github.com/apache/apisix/blob/master/ci/pod/opa/example.rego
- https://github.com/apache/apisix/blob/master/apisix/plugins/opa.lua#L23-L44

## Apache APISIX proxy Static files

Q: https://github.com/apache/apisix/issues/5792 and https://github.com/apache/apisix/discussions/5789#discussioncomment-1802733
A: The OpenResty system does not have dynamic capabilities for static content by default, so we can use Nginx custom configuration to make Location proxy static resources. https://apisix.apache.org/docs/apisix/customize-nginx-configuration/

## Plugin Flexibility

https://github.com/apache/apisix/discussions/5722

## Ingress Controller

- https://github.com/apache/apisix-ingress-controller/pull/638

**Comment by testn** [comment]
any updates re: static file proxy?

**Reply by juzhiyuan** [reply]
Hey, do you mean serving static files like NGINX?

If yes, you can continue using nginx related configurations and inject them via APISIX's config.yaml
